content/administration/maintain/hosting_changes.rst

```diff
@@ -48,13 +48,34 @@ From Odoo Online to Odoo.sh
 From Odoo.sh to Odoo Online
 ===========================
 
-.. warning:: Odoo Online is not compatible with **non-standard apps**.
+#.  Uninstall all the **non-standard apps**: test it in a staging build first, then do it in your
+    production build.
+#.  **If you have time constraints, contact us earlier to schedule the transfer.**
+#. `Create a support ticket <https://www.odoo.com/help>`_ and include the following:
 
-1.  Uninstall all the **non-standard apps**: test it in a staging build first, then do it in your production build.
-2.  **If you have time constraints, contact us earlier to schedule the transfer.**
-3. `Create a support ticket <https://www.odoo.com/help>`_ and include your subscription number and the URL you want to use for your database (e.g.: my-company.odoo.com).
-4.  We will make sure your database is compatible and upload it to our cloud. In case of technical issues, we will get in touch with you.
-5.  It's done!
+   - Your subscription number
+   - The URL you want to use for your database (e.g.: `example.odoo.com`)
+   - Which brand you want to migrate
+   - In which region you want to be hosted (select the one closest to your users to reduce latency):
+
+     - Americas
+     - Europe
+     - Asia
+
+   - Which user(s) will be the administrator(s) (they must have an odoo.com account)
+   - When (and in which timezone) you want the database to be up and running (see note below)
```

Comment:
I took off the parenthesis -just bc I don't like it here and think it looks better as a note- (same with see as note)
```suggestion
   - Which user(s) will be the administrator(s)
   - When (and in which timezone) you want the database to be up and running
```